### ISSUE TITLE
[Feature Request] Don't save at every step

### ISSUE THREAD

**Issue by ljleb** [Xerxemi/sdweb-auto-MBW#7]
Correct me if I'm wrong, but saving a checkpoint on disk at every step of adjusting merge ratios slows down the learning process by a lot IIUC. I think every `n` iterations should be treated as an epoch, i.e. it should be possible to save multiple models over time, but only save one intermediate model after a specified number of iterations.

Is it okay with you if I open a PR for this?

**Comment by Xerxemi** [comment]
You're right about that, but actually, it saves the model to disk to be able to use it to generate but doesn't keep it, it's a limitation of the current code. Xynon is looking into swapping the backend for either runtime's or supermerger's but both of us are pretty busy rn, and as such there is no eta.

Saving models as epochs wouldn't be useful, as all past final and test merges are saved to history already.

**Comment by ljleb** [comment]
Ah makes sense, as we know the original models we only need to save the receipe. simple and effective.
